src/core/stream.js

```diff
@@ -739,15 +739,16 @@ var PredictorStream = (function PredictorStreamClosure() {
     var pos = bufferLength;
     var i;
 
-    if (bits === 1) {
+    /* Optimized version of the loop in the "else"-branch
+    for 1 bit-per-component and 1 color TIFF images */
```

Comment:
Thanks for fixing this; however since you moved the comment inside of the `if` block you also need to indent it with two more spaces, i.e. like so:
```javascript
if (bits === 1 && colors === 1) {
  // Optimized version of the loop in the "else"-branch
  // for 1 bit-per-component and 1 color TIFF images.
  for (i = 0; i < rowBytes; ++i) {
    ...
```